### ISSUE TITLE
Browseable public activities should pass accessibility tests

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#2831]
If an activity does not pass all the accessibility checks, then when it is made public:
- there should be a warning saying it won't be browsable until accessibility issues are fixed, and
- it shouldn't actually be browsable.

One unresolved question: should it 
- upgrade accessibility warnings to errors and check for errors, or
- check for any accessibility warning?

At least for now, not all accessibility warnings get upgraded to errors. But, it is also a good idea to not let an activity with any errors be browsable.
